Simplenote/Classes/Simperium+Simplenote.swift

```diff
@@ -5,13 +5,34 @@ import Foundation
 //
 extension Simperium {
 
-    /// Notes Bucket
+    /// Bucket: Account
+    /// - Note: Since it's **dynamic** (InMemory JSON Storage), we don't really have an Account class
+    ///
+    var accountBucket: SPBucket {
+        bucket(forName: SimplenoteConstants.accountBucketName)
```

Comment:
What do you think about moving `SimplenoteConstants.accountBucketName` to this file? I feel it belongs here and not in the app-wide constants :-)

Reply:
You're definitely right, I never liked that!!